Mathlib/Analysis/NormedSpace/AffineIsometry.lean

```diff
@@ -877,7 +877,7 @@ namespace AffineSubspace
 /-- An affine subspace is isomorphic to its image under an injective affine map.
 This is the affine version of `Submodule.equivMapOfInjective`.
 -/
-@[simps]
+@[simps linear, simps! toFun]
```

Comment:
`simps!` tries to unfold all fields equally and goes too far, compared to what exists, on the `linear` field. 

Generally, since we no longer insert the extra eta expansion when filling the parent classes, `simps` doesn't find a constructor for those fields anymore. It then asks for `simps!`.